### PR TITLE
add key-renew-period parameter to helm chart

### PR DESCRIPTION
**Pull request by ismailyenigul**
**Description of the change**

**Benefits**
add key-renew-period parameter to helm chart

### CODE REVIEW

File: helm/sealed-secrets/values.yaml [hunk 1]
```diff
@@ -105,6 +105,10 @@ tolerations: []
 ##
 updateStatus: true
 
+## @param keyrenewperiod Specifies key renewal period. Default 30 days
+##
+keyrenewperiod: "720h30m"
```
Comment (juan131) [inline helm/sealed-secrets/values.yaml:110]: By default, the parameter shouldn't be set so the flag isn't passed, relying on the controller binary default value:
```suggestion
## @param keyrenewperiod Specifies key renewal period. Default 30 days
## e.g
## keyrenewperiod: "720h30m"
##
keyrenewperiod: ""
```

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks for this great PR! Please check my comments

**Review by alvneiayu — CHANGES_REQUESTED** [review]
Could you please execute the [readme-generator](https://github.com/bitnami-labs/readme-generator-for-helm) tool to automatically regenerate the parameters tables in README.md?  Let me know if you have any problem and I will help you. Thanks in advance and really good work.

**Review by juan131 — APPROVED** [review]
Awesome! LGTM

**Review by alvneiayu — APPROVED** [review]
LGTM, thanks a lot

**Pull request merged**